Tutorials/Connecting-Umbraco-Forms-and-Zapier/index.md

```diff
@@ -0,0 +1,156 @@
+
+## Example: Sending data to Google Sheets
+
+As a “Do this…” action you want to send data to Google Sheets. Before doing so in your Zap you need to ensure that you have created a Google Sheet to receive the data. You also need to connect your Zapier account to your Google Sheets account.
+Once you have done these steps it is time to finish the setup of the Zap.
+
+First thing to do is find the “Google Sheets” app in the “Do this…” action.
+
+![Zapier Google Sheets App](images/zapierGoogleSheetsApp.png)
+
+Now choose the Action event “Create a spreadsheet row” and continue.
+
+![Zapier Google Sheets Action Event](images/zapierGoogleSheetsActionEvent.png)
+
+Now you need to choose the Google Sheets account you want to connect to. If you have not set this up yet, you will be prompted to do so.
+Once connected you choose that account and click continue.
+
+![Zapier Google Sheets Account](images/zapierGoogleSheetsAccount.png)
+
+Now you can choose which Google Drive to use, find the spreadsheet and choose the worksheet that you want to send the data to. After doing so, you will get a list of possible fields that you can post your data to.
+
+![Zapier Google Sheets Possible Fields](images/zapierGoogleSheetsPossibleFields.png)
+
+The fields showing are all columns in your spreadsheet that have a name in row 1. To map the input data to the different fields in the spreadsheet you now click on the “Type or insert…” field and choose which data to put in that field.
+
+![Zapier Google Sheets Webhook Data](images/zapierGoogleSheetsWebhookData.png)
+
+The data that was caught by the webhook will all be dynamic and you will want to pick the fields here. That way, when a new entry comes in, the field will dynamically insert the data they submitted for that field.
+
+![Zapier Google Sheets Data Mapped](images/zapierGoogleSheetsDataMapped.png)
+
+Once you have mapped all of the fields it is time to click “continue” and test if the data is sent correctly to our spreadsheet.
+
+![Zapier Google Sheets Test](images/zapierGoogleSheetsTest.png)
+
+Click “Test & Continue” and wait for it to process. Once it is done, go to the spreadsheet you created and see if the data is there.
+
+![Google Sheets Data](images/googleSheetsData.png)
+
+Tada! Now your Zap is ready and can be activated to automatically add form entries to Google Sheets. To activate the Zap you go back to your Zap and change the toggle from “OFF” to “ON”. Now it will be waiting for new form entries and be ready to send them to Google Sheets.
+
+## Connecting multiple Umbraco Forms to Zapier
+
+If you want to connect multiple forms to Zapier you can follow the above steps to do so.
+Be sure to add individual Zaps to each form if the data being sent is different or if you want to use different actions for the data.
+You can reuse the same Zap and webhook URL if all data sent to Zapier is formatted in the same way. Otherwise errors will occur.
```

Comment:
```suggestion

You can reuse the same Zap and webhook URL if all data sent to Zapier is formatted in the same way. Otherwise errors will occur.
```